rule/testdata/43-module-load.rules.golden.yml

```diff
@@ -0,0 +1,164 @@
+# Linux dd81ed8942a5 4.9.27-moby #1 SMP Thu May 11 04:01:18 UTC 2017 x86_64 GNU/Linux
+# auditctl version 2.6.7
+rules:
+- flags: -w /tmp/audit-test/sbin/insmod -p x -k modules
+  bytes: !!binary |
```

Comment:
That is best summed up at https://github.com/elastic/go-libaudit/pull/7/files#diff-6ec0fa0617d1d30c16ba468abb5a1dcaR24

> // TestUpdateGoldenData generates new X.rules.golden.yml files the given
// X.rules files. The .rules files contain rules that are installed to the
// kernel using auditctl (auditd package must be installed). Then the binary
// representation of the rule from the kernel is requested and stored. Each
// rules and blob is stored in the YAML file as a test case.
//
// The kernel version and auditctl version used to generate the golden data
// are stored as comments in the YAML file header.
func TestUpdateGoldenData(t *testing.T) {